setup.py

```diff
@@ -72,5 +72,9 @@ def get_version():
             "docker>=7.1.0",
             "pytest-mock>=3.14.1",
         ],
+        "docs": [
```

Comment:
I had to add docs as a separate list of packages, because dependencies wouldn't resolve successfully unless it was installed after "dev"